src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,41 +1,68 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
```

Comment:
Ensure that the 'getUser' function is correctly implemented in the '../../api' file and returns the expected user data.